pkgs/servers/caddy/default.nix

```diff
@@ -1,8 +1,16 @@
 { lib, buildGoModule, fetchFromGitHub, nixosTests }:
-
+let
+  version = "2.4.6";
+  dist = fetchFromGitHub {
+    owner = "caddyserver";
+    repo = "dist";
+    rev = "v${version}";
+    sha256 = "sha256-EXs+LNb87RWkmSWvs8nZIVqRJMutn+ntR241gqI7CUg=";
+  };
+in
 buildGoModule rec {
```

Comment:
detail: since you extracted version, I'm not sure we need rec anymore. it's optional of course, but might be good to remove it.

Reply:
Good catch. Thanks!